### PR TITLE
Third set of TestBuild* CI enabling for Windows

### PR DESCRIPTION
**Pull request by darstahl**
Signed-off-by: Darren Stahl darst@microsoft.com

### CODE REVIEW

**Comment by darrenstahlmsft** [comment]
This PR is dependent on #19790, which is why windowsTP4 is currently failing.

File: integration-cli/docker_cli_build_test.go [hunk 1]
```diff
@@ -1837,13 +1837,22 @@ func (s *DockerSuite) TestBuildWithInaccessibleFilesInContext(c *check.C) {
...
 	}
 	name := "testbuildforcerm"
-	ctx, err := fakeContext("FROM scratch\nRUN true\nRUN thiswillfail", nil)
+
+	var baseImage string
+	if daemonPlatform == "windows" {
+		baseImage = "windowsservercore"
+	} else {
+		baseImage = "scratch"
+	}
```
Comment (thaJeztah) [inline integration-cli/docker_cli_build_test.go]: Wondering if we should also use some helper method for this, similar to https://github.com/docker/docker/pull/19767. No need to do that in this PR though

Comment (darrenstahlmsft) [inline integration-cli/docker_cli_build_test.go]: I agree, I was just starting to look at where to put that. That PR is exactly what I needed. Thanks

Comment (thaJeztah) [inline integration-cli/docker_cli_build_test.go]: You're welcome!

**Comment by jhowardmsft** [comment]
Please do not merge yet. See https://github.com/docker/docker/pull/19847. CI should be re-run after 19847 is merged.

**Comment by icecrime** [comment]
Are we waiting for some of this code to be factored out before merging?

**Comment by jhowardmsft** [comment]
@icecrime Yes.

**Comment by darrenstahlmsft** [comment]
As soon as #19985 is merged I can rebase this PR and rerun CI

**Comment by cpuguy83** [comment]
Is this one ready?

**Comment by darrenstahlmsft** [comment]
As soon as CI passes this is good to go. Due to the new tests being enabled, I would like to see windowsTP4 green at the very least.

**Comment by darrenstahlmsft** [comment]
CI is green :+1:

**Comment by cpuguy83** [comment]
LGTM

**Comment by vdemeester** [comment]
LGTM 🐮

**Pull request merged**